### PR TITLE
Implemented EmailClient with "dummy" sender-email. Givers now receive an confirmation on registering

### LINKED ISSUE #106: Check out different Email senders for ASP.NET Core

### PR DESCRIPTION
**Pull request by Sommerli**
Closes #106 
See Issue for multiple links that helped me solve this implementation.

Add possibility to send mails via EmailClient. Body renders HTML.

Added EmailClient to AdminController(unused) and GiverController. In GiverController new registrations now recive a confirmation-mail upon signup.

Made two configuration-sections. One for a dummy mail and one for the actual mail we will use in production..

### CODE REVIEW

File: GiEnJul/Clients/EmailClient.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        public EmailClient(ISettings settings, IWebHostEnvironment env, ILogger log)
+        {
+			_mailSettings = settings.MailSettings;
+			_env = env;
+			_log = log;
+        }
+
+        public async Task SendEmailAsync(string toMail, string toName, string subject, string body)
+		{
```
Comment (RonHaland) [inline GiEnJul/Clients/EmailClient.cs:30]: This file has strange formatting

File: GiEnJul/appsettings.json [hunk 2]
```diff
@@ -7,5 +7,19 @@
...
+    "Password": "kulmailkonto2021",
+    "Host": "smtp.gmail.com",
+    "Port": 587
+  },
+  "OneMailSettings": {
+    "Mail": "noreply@gienjul.no",
+    "DisplayName": "Gi En Jul",
+    "Password": "password",
+    "Host": "send.one.com",
+    "Port": 465
```
Comment (RonHaland) [inline GiEnJul/appsettings.json:23]: I think there should only be one MailSettings section.
The commit should include local settings, that can be replaced in the deploy pipeline.
We could consider using Secrets in AzureKeyVault for the passwords.

Maybe make a note of the other settings in another issue?

**Review by RonHaland — CHANGES_REQUESTED** [review]

File: GiEnJul/appsettings.json (hunk 2 above)
Comment (Sommerli) [reply GiEnJul/appsettings.json:23]: Created #241. The current mail should probably not be used in the future. Maybe prod and dev could use the same mail(noreply@gienjul.no)?

**Comment by Sommerli** [comment]
With current implementation, papercut is needed for local testing of mailclient. 

https://github.com/ChangemakerStudios/Papercut-SMTP/releases
Follow installation guide and run the program to send mails locally.

@elisabethnetland @TheseusDeus @joffe97

**Review by RonHaland — APPROVED** [review]

**Pull request merged**